app/assets/javascripts/application.js

```diff
@@ -75,8 +75,8 @@ function onContentLoad() {
     // return false for actual disabled links, this is
     // required in case the link was "enabled" after the
     // function has registered.
-
-    return !this.disabled;
+    var isDisabled = $(this).attr('disabled') === 'disabled';
```

Comment:
I think Tomer meant this:
```suggestion
    var isDisabled = $(this).prop('disabled');
```

Reply:
thanks though `.prop` seem to return `undefined` for some reason, so lets use:
`!$(this).attr('disabled') === 'disabled';` instead.

Reply:
Looks like this bug is affecting links that are actually incorrectly setting the `disabled` attribute to `disabled`, which we apparently have quite a lot of (try searching for `attr('disabled', 'disabled')`). :crying_cat_face: tbh i'm not sure why @ohadlevy made this change in https://github.com/theforeman/foreman/pull/6529/files#diff-a9c3bd311eab80c9ebe6a69830f9ad02R65

Reply:
@laviro and will it work even if I set the link like `.attr('disabled', true)`?

Reply:
fyi, https://github.com/theforeman/foreman/pull/3241 was the last attempt to fix this properly :\

Reply:
> Looks like this bug is affecting links that are actually incorrectly setting the `disabled` attribute to `disabled`, which we apparently have quite a lot of (try searching for `attr('disabled', 'disabled')`).  tbh i'm not sure why @ohadlevy made this change in https://github.com/theforeman/foreman/pull/6529/files#diff-a9c3bd311eab80c9ebe6a69830f9ad02R65

I would say it was meant to use the prop. For that you don't need jquery...

Reply:
@ezr-ondrej the event listener will get called only on element like: `a[disabled="disabled"]`

Reply:
@tbrisker @ezr-ondrej  I agree setting the `disabled` attribute on an `<a>` tag is weird, but as long as this function properly checks for it, it seems to work.  I made a [codepen](https://codepen.io/jeremy646/pen/aboGWLJ?editors=1111) to verify this.